examples/Wiper/Wiper.ino

```diff
@@ -0,0 +1,39 @@
+/*
+ Using a servo as a car Wiper, activated by switch.
+
+ modified on 19 March 2020
+ by Durgesh Pachghare
+*/
+
+#include <Servo.h>
+
+Servo myservo;  // create servo object to control a servo
+
+int button_pin = 2;  // digital pin used to connect the button input
+int val;    // variable to read the value from the analog pin
+int startpos = 10;      //angle from which the servo wiper starts
+int endpos = 145;       //angle till the servo wiper rotates
+
+void setup() 
+{
+  myservo.attach(9);  // attaches the servo on pin 9 to the servo object
+  pinMode(button_pin,INPUT);    //declares the pin as digital input

```

Comment:
If you're going to use `INPUT` mode, then you should make it clear that a pull-down resistor is required. Better would be to use `INPUT_PULLUP` mode.

Reply:
Okay, I have added INPUT_PULLUP and added a necessary description.